packages/extension-api/src/extension-api.d.ts

```diff
@@ -2280,6 +2293,17 @@ declare module '@podman-desktop/api' {
     export function listVolumes(): Promise<VolumeListInfo[]>;
     export function createVolume(options?: VolumeCreateOptions): Promise<VolumeCreateResponseInfo>;
     export function deleteVolume(volumeName: string, options?: VolumeDeleteOptions): Promise<void>;
+
+    export function createPod(
+      podOptions: PodCreateOptions,
+      selectedProvider?: ProviderContainerConnectionInfo | ContainerProviderConnection,
```

Comment:
`provider` should be inside PodCreateOptions (maybe my previous comment has been lost)

Reply:
ops i missed it. Done :+1